src/main/java/org/cyclops/integrateddynamics/core/recipe/type/RecipeFacadeSqueezeMechanical.java

```diff
@@ -0,0 +1,26 @@
+package org.cyclops.integrateddynamics.core.recipe.type;
+
+import org.cyclops.integrateddynamics.RegistryEntries;
+
+import net.minecraft.core.NonNullList;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.Ingredient;
+import net.minecraft.world.item.crafting.RecipeSerializer;
+import net.minecraftforge.fluids.FluidStack;
+
+public class RecipeFacadeSqueezeMechanical extends RecipeMechanicalSqueezer {
+	
+    public RecipeFacadeSqueezeMechanical(ResourceLocation id, Ingredient inputIngredient, NonNullList<IngredientChance> outputItems, FluidStack outputFluid, int duration) {
+        super(id, inputIngredient, outputItems, outputFluid, duration);
+    }
+
+    public NonNullList<IngredientChance> assemble(ItemStack inputItem) {
```

Comment:
Does this one actually get called anywhere?
If I'm correct, this is a new method (no override).
But it doesn't seem to get called anywhere.